src/game-engine/GameEngineA.ts

```diff
@@ -41,13 +41,13 @@ export default class GameEngineA {
   }
 
   state: State.PlayerAState;
- 
+
   constructor(state) {
     this.state = state;
   }
 
   receivePosition(positionReceived: Position) {
-    switch(positionReceived.constructor) {
+    switch (positionReceived.constructor) {
```

Comment:
General point: I know we have this in our code a lot but I'm not convinced it is safe anymore (due to changes in names when minifying). We should probably figure out definitively if it is safe and move away from it if not.